docs/source/conf.py

```diff
@@ -66,8 +66,6 @@ def get_version():
 # Breathe configuration
 breathe_projects = {
```

Comment:
Not super sure on this change - what should I be looking for with the documentation as part of this?